static/app/components/forms/selectAsyncControl.tsx

```diff
@@ -13,7 +13,7 @@ type Result = {
   label: string;
 };
 
-type Props = {
+export type Props = {
```

Comment:
Instead of exporting this, you can import `SelectAsyncControl` and use `React.ComponentProps<typeof SelectAsyncControl>`